automation/check-patch.e2e-kubemacpool-functests.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash -xe
+
+# This script should be able to execute KubeMacPool
+# functional tests against Kubernetes cluster with
+# CNAO built with latest changes, on any
+# environment with basic dependencies listed in
+# check-patch.packages installed and docker running.
+#
+# yum -y install automation/check-patch.packages
+# automation/check-patch.e2e-kubemacpool-functests.sh
+
+teardown() {
+    echo "make cluster-down"
+    rm -rf "${TMP_COMPONENT_PATH}"
+}
+
+main() {
+    # Setup CNAO and artifacts temp directory
+    source automation/check-patch.setup.sh
+    cd ${TMP_PROJECT_PATH}
+
+    # Spin-up ephemeral cluster with latest CNAO
+    # this script also exports KUBECONFIG, and fetch $COMPONENT repository
+    export COMPONENT="kubemacpool"
```

Comment:
We don't need to export this, we can instead just call following line with `COMPONENT='kubemacpool' source automation/components-functests.setup.sh`.

That way we make sure the export does not spoil the rest of the scope or other called scripts.

Reply:
Done